File: app/src/main/java/ru/eva/oriokslive/models/schedule/Room.java
package ru.eva.oriokslive.models.schedule;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import io.realm.RealmObject;

public class Room extends RealmObject {
    @SerializedName("Code")
    @Expose
    private String code;
    @SerializedName("Name")
    @Expose
    private String name;

    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

}

File: app/src/main/java/ru/eva/oriokslive/network/entity/schedule/Room.kt
package ru.eva.oriokslive.network.entity.schedule

import androidx.room.ColumnInfo
import com.google.gson.annotations.SerializedName

data class Room(
    @SerializedName("Code")
    @ColumnInfo(name = "code")
    val code: String,
    @SerializedName("Name")
    @ColumnInfo(name = "name")
    val name: String,
)
